package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Just curious, what uses this?

Reply:
that, I need to take out - that is from when I was following a different tutorial to make auth0 changes - thanks for catching!